examples/sdk-usage/src/main/java/io/opentelemetry/sdk/example/ConfigureTraceExample.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.example;
+
+import io.opentelemetry.common.AttributeValue;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.Sampler;
+import io.opentelemetry.sdk.trace.Samplers;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.TracerSdk;
+import io.opentelemetry.sdk.trace.TracerSdkProvider;
+import io.opentelemetry.sdk.trace.config.TraceConfig;
+import io.opentelemetry.trace.Link;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceId;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+class ConfigureTraceExample {
+
+  // Class that prints in the console custom information about the spans for the sake of the
+  // example. For more details about the default SpanProcessor, refer to
+  // ConfigureSpanProcessorExample.java
+  private static class MyProcessor implements SpanProcessor {
```

Comment:
Just use the simplespanprocessor + loggingexporter?

Reply:
I wanted to showcase the possibility to create your own span processor with this class. Furthermore, with this class the attributes are pretty printed while with the loggingexporter they would be embedded with other span information.

Reply:
Then split it into one example to configure the SDK using TraceConfig + Views for metrics when they will be available. Second example to create your own SpanProcessor. I think most of the people are interested in the first example.